rust/processor/src/utils/util.rs

```diff
@@ -413,6 +414,41 @@ pub fn get_name_from_unnested_move_type(move_type: &str) -> &str {
     t.last().unwrap()
 }
 
+// pub fn get_size_in_kbs_of_vectors<T>(vectors: &[Vec<T>]) -> usize {
+//     vectors
+//         .iter()
+//         .fold(0, |acc, v| acc + mem::size_of_val(&v[0]) * v.len())
+// }
+
+// pub fn calculate_size_in_kbs<T>(vector: Vec<T>) -> f64 {
+//     let total_size_in_bytes = mem::size_of_val(&vector[0]) * vector.len();
+//     let total_size_in_kb = total_size_in_bytes as f64 / 1024.0;
+//     total_size_in_kb
+// }
+
+// pub fn calculate_size_in_kbs(vectors: Vec<Box<dyn std::fmt::Debug>>) {
+//     vectors
+//         .iter()
+//         .fold(0.0, |acc, v| acc + mem::size_of_val(&v[0]) * v.len())
+// }
+
+#[macro_export]
+macro_rules! calculate_size_in_kbs {
```

Comment:
provide some reference on where this comes from? Is this chatgpt or stack overflow?